bin/bookkeeper-cli

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
+#
+#/**
+# * Copyright 2007 The Apache Software Foundation
+# *
+# * Licensed to the Apache Software Foundation (ASF) under one
+# * or more contributor license agreements.  See the NOTICE file
+# * distributed with this work for additional information
+# * regarding copyright ownership.  The ASF licenses this file
+# * to you under the Apache License, Version 2.0 (the
+# * "License"); you may not use this file except in compliance
+# * with the License.  You may obtain a copy of the License at
+# *
+# *     http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# */
+
+BINDIR=`dirname "$0"`
+BK_HOME=`cd $BINDIR/..;pwd`
+
+DEFAULT_CONF=$BK_HOME/conf/bk_server.conf
```

Comment:
in bookkeeper/conf folder there isn't bk_server.conf, it is in bookkeeper/bookkeeper-server/conf folder

so getting following error -

Failed to load configuration file '/Users/cguttapalem/Workspace/Community/bookkeeper/conf/bk_server.conf' : Unable to load the configuration from the URL file:/Users/cguttapalem/Workspace/Community/bookkeeper/conf/bk_server.conf

Reply:
yes. that is done intentionally at this PR. was planning to move the bin/ and conf/ to root directory after we complete the new bookkeeper CLI. otherwise we will have two bk_server.conf in place, which make things a bit hard to keep it in sync.

for testing purpose, you can manually copy bookkeeper-server/conf/bk_server.conf to conf.

If you feel strong that we need to copy bookkeeper-server/conf/bk_server.conf  right now, we don't have to wait until new bookkeeper cli is completed, I am fine to copy the file as well.